<?xml version="1.0" encoding="UTF-8"?>
<launch>
	<arg name="image_path" default="$(find tb_brngup)/heightimages/d2.png"/>
	<arg name="radius_surroundcloud" default="25.0"/>
	<arg name="image_elevation_max"  default="50.0"/>
	<arg name="limit_dhdng" default="1.5"/>
	<arg name="limit_dincl"  default="1.0"/>
	<arg name="sensor_range"  default="50.0"/>

	<node name="tb_simulate_img2sensor_node"  pkg="tb_simulate_perception" type="tb_simulate_img2sensor_node" output="screen" respawn="true">
		<param name="image_path" 											 type="string" value="$(arg image_path)" />
		<param name="sensor_range_of_view"	 				   type="string" value="$(arg sensor_range)" />
		<param name="sensor_angle_of_view_vertical"	   type="double" value="$(arg limit_dincl)" />
		<param name="sensor_angle_of_view_horisontal"	 type="double" value="$(arg limit_dhdng)" />
		<param name="normalized_image_zmax"	 					 type="double" value="$(arg image_elevation_max)" />
		<param name="image_is_normalized"	 				 		 type="bool"   value="true" />
	</node>

	<!--node name="tb_img2sensor_node"   pkg="tb_simulate_perception" type="tb_img2sensor_node.py" output="screen"		respawn="true">
		<param name="image_path" type="string" value="$(arg image_path)" />
		<param name="radius_surroundcloud" type="double" value="$(arg radius_surroundcloud)" />
		<param name="image_elevationmax" type="double" value="$(arg image_elevation_max)" />
		<param name="limit_dhdng" type="double" value="$(arg limit_dhdng)" />
		<param name="limit_dincl" type="double" value="$(arg limit_dincl)" />
	</node-->
</launch>
